src/messages/TextMessage.vue

```diff
@@ -1,37 +1,47 @@
 <template>
-  <div class="sc-message--text" :style="messageColors">
+  <div class="sc-message--text" v-bind:class="{ 'confirm-delete' : deleteConfirmation == true }" :style="messageColors">
     <template>
       <div class="sc-message--toolbox" :style="{background: messageColors.backgroundColor}">
-        <button
-          v-if="showEdition && me && message.id != null && message.id != undefined"
-          :disabled="isEditing"
-          @click="edit"
-        >
-          <IconBase :color="isEditing ? 'black' : messageColors.color" width="10" icon-name="edit">
-            <IconEdit />
-          </IconBase>
+        <button v-if="showEdition && me && message.id != null && message.id != undefined" @click="edit" :disabled="isEditing">
```

Comment:
Strict inequality (!==) should be used but just `... && message.id"` may be enough since you seem to be filtering for truthy values.